packages/core/test/CSL/coreToCsl.test.ts

```diff
@@ -40,15 +40,17 @@ describe('coreToCsl', () => {
       expect(value).toBeInstanceOf(CSL.Value);
     });
   });
-  it('tokenMap', () => {
-    const multiasset = coreToCsl.tokenMap(txOut.value.assets!);
-    expect(multiasset).toBeInstanceOf(CSL.MultiAsset);
-    expect(multiasset.len()).toBe(2);
-    for (const [assetId, expectedAssetQuantity] of txOut.value.assets!.entries()) {
-      const { scriptHash, assetName } = Asset.util.parseAssetId(assetId);
-      const assetQuantity = BigInt(multiasset.get(scriptHash)!.get(assetName)!.to_str());
-      expect(assetQuantity).toBe(expectedAssetQuantity);
-    }
+  describe('tokenMap', () => {
+    it('inserts a single multiasset per asset policy', () => {
+      const multiasset = coreToCsl.tokenMap(txOut.value.assets!);
+      expect(multiasset).toBeInstanceOf(CSL.MultiAsset);
+      expect(multiasset.len()).toBe(3);
```

Comment:
That would be a little clearer as keys of MultiAsset is ScriptHashes which is what we want to test here. Not sure if there's any difference functionally, but I think more explicit is better:

```suggestion
      expect(multiasset.keys().len()).toBe(3);
```

Reply:
Done in 37e9d008328ca1f17efc8e93bb8eade91317762f